build.gradle

```diff
@@ -10,7 +8,7 @@ buildscript {
 
 plugins {
     id "java"
```

Comment:
We might want to migrate to java-library instead of plain java so we support the API configuration.